.github/workflows/main.yml

```
@@ -0,0 +1,32 @@
+name: Vapor OAuth
+
+on:
+  push:
+    branches: [ "main" ]
+    tags:
+      - '*'
+  pull_request:
+    branches: [ "main" ]
```

Let's change this to any PR
```suggestion
    branches: *
```